src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -1047,8 +1047,9 @@ private InstructorFeedbackResultsQuestionTable buildQuestionTableAndResponseRows
         }
         
         String studentEmail = student == null ? null : student.email;
-        String statisticsTable = questionDetails.getQuestionResultStatisticsHtml(responses, question, studentEmail,
-                                                                                 bundle, viewType.toString());
+        String statisticsTable =
+                questionDetails.getQuestionResultStatisticsHtml(responses, question, studentEmail,
+                                                                bundle, viewType.toString());
```

Comment:
Maybe keep the method call on the first line, and put the arguments on the second line